frameworks/V/pico.v/pico.v.dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM debian:sid
+FROM syou/v-dev:0.1
```

Comment:
We prefer official bases so we can see in the dockerfile here all the steps taken to get your framework built and running. Also, it helps with the docker cache and testing speed in our main environment.

Reply:
@nbrady-techempower , Framework is built under that in pico.v.dockerfile, docker image is the compiling V language itself with some commits picked from master.

Reply:
But just let me know if you insist @nbrady-techempower . I could just go ahead and copy paste parent docker commands.

Reply:
This is ok for now, then. If part of updating the framework means you'll be updating this base, when you do that, can you please copy and paste the commands in a comment here for quick review. We want to make sure that it's not always pulling the latest commit, etc. There's just some things we need to audit.

Reply:
here is the base docker file on docker hub - https://hub.docker.com/r/syou/v-dev/dockerfile
and it is tagged as 0.1, so unless there is V language update, there will be no addition pull - https://hub.docker.com/r/syou/v-dev/tags

Reply:
> when there is framework update, line 7 or 8 or 9 in pico.v.dockerfile will be updated to specific tag. like, git checkout v0.0.4 -> git checkout v0.0.5

Yes, this is my point. We want to make sure that the base you're pulling from isn't pulling the latest from master inside the docker container; we clear the docker cache every few runs. When I talk about the docker cache, I mean it in the context of a single run, such that a lot of frameworks share the same base like `buildpack-deps:xenial` and the first few lines are often the same, like apt updating.